multimedia/lmms/Portfile

```diff
@@ -79,6 +74,14 @@ configure.args-append   -DFLTK_FLUID_EXECUTABLE=${prefix}/bin/fluid \
                         -DWANT_STK=ON \
                         -DWANT_VST=OFF
 
+if {${os.platform} eq "darwin" && ${configure.build_arch} in [list ppc ppc64]} {
+    # See: https://github.com/jackaudio/jack2/issues/950
+    configure.args-replace \
+                        -DWANT_JACK=ON -DWANT_JACK=OFF
+} else {
+    depends_lib-append  port:jack
+}
```

Comment:
This feels a little backwards to me. I would instead change `-DWANT_JACK=ON` to `-DWANT_JACK=OFF` in the `configure.args-append` section above, and then change this section to:
```suggestion
# See: https://github.com/jackaudio/jack2/issues/950
if {${os.platform} ne "darwin" || ${configure.build_arch} ni [list ppc ppc64]} {
    configure.args-replace \
                        -DWANT_JACK=OFF -DWANT_JACK=ON
    depends_lib-append  port:jack
}
```
And of course this block (doing something based on `build_arch`) is another impediment to the port getting a universal variant.

Reply:
@ryandesign I will make the suggested change today, thank you.

For universal: well, we can safely assume no one gonna build `ppc+x86`, since, to begin with, it is broken in Macports for numerous ports, it will be simply impossible to get to the point of _trying_ to build `lmms` that way.

From the upstream GCC ticket it follows that the bug breaking `jack` affects `i386` and `ppc` but not 64-bit archs. So it in fact might build on `ppc64`. But we can prioritize universal and disable `jack` on powerpc (`i386` is irrelevant since Macports gonna use `clang`, right?).